server/src/stateManagement.mjs

```diff
@@ -739,6 +714,29 @@ function createUuid(){
   return uuidv4();
 }
 
+/* 
+* @function:doesUserExist()
+* @Description: check if the user/appId exists
+* @Params:userId,users
+* @Return: JSON object {isSuccess: true/false, msg : ``}
+*/
+
+function doesUserExist(users,userId){
+  const { user, appId } =parseUser(userId)
+  //iterating over list of users in state to ensure duplicate user/appId exist
+  for(var key in users){
+    const { user: existingUser, appId : existingAppId } =parseUser(users[key])
+    if (user && existingUser && existingUser==user){
```

Comment:
Very minor: Generally speaking, it's better to use triple equals in JavaScript instead of double equals. Not necessary to change as part of this PR, I don't think.